common/SigUtil.cpp

```diff
@@ -243,6 +251,11 @@ namespace SigUtil
             Log::signalLog(" Fatal signal received: ");
         Log::signalLog(signalName(signal));
 
+        if(dumpUnoCommandsInfoFn != nullptr)
```

Comment:
Our code style is to leave a space after keywords like 'if'.

Reply:
Ok Tor got it . Will update the changes